validator/validator-main.protoascii

```diff
@@ -6710,6 +6710,9 @@ attr_lists: {
   attrs: { name: "amp-access-show" }
   attrs: { name: "amp-access-style" }
   attrs: { name: "amp-access-template" }
+  # TODO(wassgha): Launch with amp-next-page 1.0
```

Comment:
Personally I don't like this approach of adding commented out validator rules that may be uncommented later. Lets just have another PR that adds them explicitly.

Reply:
Fixed